### PR TITLE
Feature/zk UI 414 create configuration overalay provider

### PR DESCRIPTION
**Pull request by JBWatenbergScality**

### CODE REVIEW

File: src/react/locations/LocationEditor.tsx [hunk 1]
```diff
@@ -34,6 +34,8 @@ import {
 import { useQueryClient } from 'react-query';
 import { queries } from '../next-architecture/domain/business/locations';
 import { useLocationAdapter } from '../next-architecture/ui/LocationAdapterProvider';
```
Comment (ChengYanJin) [inline src/react/locations/LocationEditor.tsx]: Magic comment: OPTION + SHIFT + O  !

**Review by ChengYanJin — APPROVED** [review]
LGTM !!

File: src/react/locations/LocationEditor.tsx (hunk 1 above)
Comment (JBWatenbergScality) [reply src/react/locations/LocationEditor.tsx]: imagination-spongebob-squarepants-gif

**Pull request merged**